docs/api/ListenerClose.md

```diff
@@ -23,7 +23,7 @@ A valid handle to an open listener object.
 
 # Remarks
 
-`ListenerClose` frees all allocated resources associated with the listener handle. If a listener has not had [ListenerStop](ListenerStop.md) called on it at the time `ListenerClose` is called, [ListenerStop](ListenerStop.md) is invoked internally. Due to this, a server application **MUST NOT** call `ListenerClose` within any callback unless it has already called [ListenerStop](ListenerStop.md) outside of any callbacks.
+`ListenerClose` frees all allocated resources associated with the listener handle. If a listener has not had [ListenerStop](ListenerStop.md) called on it at the time `ListenerClose` is called, [ListenerStop](ListenerStop.md) is invoked internally. `ListenerClose` may be called on any Listener callback. A server application **MUST NOT** call `ListenerClose` within any callback except for `QUIC_LISTENER_EVENT_STOP_COMPLETE`, or unless it has already received the `QUIC_LISTENER_EVENT_STOP_COMPLETE` event.
```

Comment:
The second part here seems to contradict the first. It says you can call `ListenerClose` without calling `ListenerStop` first. But then it says you cannot call `ListenerClose` unless you have received the `QUIC_LISTENER_EVENT_STOP_COMPLETE` event.

Reply:
Fixed